src/handlers/motions/motionFinalized/helpers.ts

```diff
@@ -243,3 +243,34 @@ export const linkPendingColonyMetadataWithColony = async (
     },
   });
 };
+
+export const updateColonyUnclaimedStakes = async (
+  colonyAddress: string,
+  motionTxHash: string,
+  motionDatabaseId: string,
+  updatedStakerRewards: StakerReward[],
+): Promise<void> => {
+  const colony = await getColonyFromDB(colonyAddress);
+  if (colony) {
+    const unclaimedMotionStake = {
```

Comment:
When the motion is finalized, all stakes are unclaimed. So we just add a new entry to the `motionsWithUnclaimedStakes` array, with the motion being finalized.